README.md

```diff
@@ -11,111 +11,58 @@ This project filters tests based on static analysis of changed files. Changed fi
 -  DeepDiff (Python package)
 -  Jinja2 (Python package)
 -  xmldiff (Python package)
--  requests (Python package, required for experiments and Github comments)
 
 ## Installation
 Fedora/RHEL:
 ```
 yum install python3 git
-pip install gitpython PyYAML deepdiff Jinja2 xmldiff requests
+pip3 install gitpython PyYAML deepdiff Jinja2 xmldiff
 git clone https://github.com/mildas/content-test-filtering
 ```
 
 **WARNING:** For analysis of Jinja files, ComplianceAsCode dependencies are required. See [ComplianceAsCode Developer Guide](https://github.com/ComplianceAsCode/content/blob/master/docs/manual/developer_guide.adoc)
 
 ## Usage
-Project has two modes how to get changes from `git`:
--  **Pull request number** - get changes from a specific [Pull request created for ComplianceAsCode project](https://github.com/ComplianceAsCode/content/pulls)
+Options how to obtain changes:
+-  **Pull request number** - get changes from a [ComplianceAsCode/content PR](https://github.com/ComplianceAsCode/content/pulls)
 -  **Branch name** - name of a branch
-```
-$ python3 content_test_filtering.py --help
```

Comment:
Why have you decided to remove this part but leave the optional arguments later on?

Reply:
The `pr` and `branch` options are described few lines above, so having the output there describing the same options is redundant.

Reply:
Thinking about it, the `positional arguments` part actually might be useful to emphasize they are needed. I'll put it back